docker/server/nginx/etc/nginx/conf.d/universe.marathon.mesos.conf

```diff
@@ -72,13 +72,25 @@ server {
     types {
       "application/vnd.dcos.universe.repo+json;charset=utf-8;version=v3" json;
     }
+    
+    if ($dcos_release_version = "1.10") {
+      rewrite ^/v3$ /repo-empty-v3.json break;
+    }
+    
+    
     rewrite ^/v3$ /repo-up-to-$dcos_release_version.json break;
   }
 
   location = /v4 {
     types {
       "application/vnd.dcos.universe.repo+json;charset=utf-8;version=v4" json;
     }
+    
+    if ($dcos_release_version != "1.10") {
+      rewrite ^/v4$ /repo-empty-v3.json break; # empty v3 = empty v4
```

Comment:
It would also make it easier to encode content-types, and (depending on how we do the rewrite logic) we could make it generic enough so that we may not need to update with every dcos-release version.